src/components/notification-panel/notification-item.js

```diff
@@ -0,0 +1,123 @@
+// Copyright (c) 2019 Uber Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import Delete from '../common/icons/delete';
+import Info from '../common/icons/info';
+import Warning from '../common/icons/warning';
+import Checkmark from '../common/icons/checkmark';
+import ReactMarkdown from 'react-markdown';
+
+const NotificationItemContent = styled.div`
+  background-color: ${props => props.theme.notificationColors[props.notification.type] || '#000'};
+  color: #fff;
+  display: flex;
+  flex-direction: row;
+  width: ${props => props.theme.notificationPanelItemWidth * (1 + Number(props.isExpanded))}px;
+  height: ${props => 
+    props.theme.notificationPanelItemHeight * (1 + Number(props.isExpanded)) 
+  }px;
+  font-size: 10px;
+  margin-bottom: 1rem;
+  padding: 1em;
+  -webkit-border-radius: 4px;
+  -moz-border-radius: 4px;
+  border-radius: 4px;
+  -webkit-box-shadow: ${props => props.theme.boxShadow};
+  -moz-box-shadow: ${props => props.theme.boxShadow};
```

Comment:
afaik the styled-components autoprefixer should handle adding these

Reply:
thank you